singer_sdk/plugin_base.py

```diff
@@ -44,7 +68,10 @@
 class PluginBase(metaclass=abc.ABCMeta):
     """Abstract base class for taps."""
 
-    name: str  # The executable name of the tap or target plugin.
+    name: str  # The executable name of the tap or target plugin. e.g. tap-foo
+    package_name: str | None = (
+        None  # The package name of the plugin. e.g meltanolabs-tap-foo
+    )
```

Comment:
Can we make this so they'll be documented in the docs site?

```suggestion
    #: The executable name of the tap or target plugin. e.g. tap-foo
    name: str

    #: The package name of the plugin. e.g meltanolabs-tap-foo
    package_name: str | None = None
```